### PR TITLE
change: [M3-7499, M3-7502, M3-7507] – UX copy updates for VPC Panel in Linode Create & Linode Add/Edit Config Dialog

### PR DESCRIPTION
**Pull request by dwiley-akamai**
## Description 📝
Update UX copy in the VPCPanel and Linode Add/Edit Config dialog

## Changes  🔄
- UX copy updates and added a docs link

### Prerequisites
- Account with proper VPC tags

### Verification steps
- VPCPanel changes can be verified in the Linode Create flow and/or the Linode Add/Edit Config dialog
  - "Learn more" is hyperlinked to the finalized (but unpublished) doc URL
  - Copy in tooltip for VPC dropdown changed
  - Copy in tooltip for "Assign a public IPv4 address for this Linode" checkbox changed
  - Copy in tooltip for "Auto-assign a VPC IPv4 address for this Linode in the VPC" checkbox changed
- Copy changes for unrecommended configuration notices in Add/Edit Config dialog (instructions for specific configurations/scenarios outlined in "Preview" section in https://github.com/linode/manager/pull/9916)

## As an Author I have considered 🤔
- [X] 👀 Doing a self review
- [X] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [X] 🤏 Splitting feature into small PRs
- [X] ➕ Adding a changeset
- [X] 🧪 Providing/Improving test coverage
- [X] 🔐 Removing all sensitive information from the code and PR description
- [X] 🚩 Using a feature flag to protect the release
- [X] 👣 Providing comprehensive reproduction steps
- [X] 📑 Providing or updating our documentation
- [X] 🕛 Scheduling a pair reviewing session
- [X] 📱 Providing mobile support
- [X] ♿  Providing accessibility support

### CODE REVIEW

**Comment by dwiley-akamai** [comment]
(depending on the exact timeline of when this is ready to be merged, I may delete the changeset and update the changelog directly before merging, since this is pointed at the release branch)

File: packages/manager/src/features/VPCs/constants.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const SUBNET_UNASSIGN_LINODES_WARNING = `Unassigning Linodes from a subne
 export const VPC_LABEL = 'Virtual Private Cloud (VPC)';
 
 export const VPC_AUTO_ASSIGN_IPV4_TOOLTIP =
-  'A VPC IPv4 is the private IP address for this Linode in the VPC.';
+  'Automatically assigns an IPv4 address as the private IP address for this Linode in the VPC.';
```
Comment (hana-linode) [inline packages/manager/src/features/VPCs/constants.ts:19]: "Automatically assign..." reads better than "Automatically assigns..."

**Review by hana-linode — APPROVED** [review]
- "Learn more" is hyperlinked to the finalized (but unpublished) doc URL
- Copy in tooltip for VPC dropdown changed
- Copy in tooltip for "Assign a public IPv4 address for this Linode" checkbox changed
- Copy in tooltip for "Auto-assign a VPC IPv4 address for this Linode in the VPC" checkbox changed
- Copy changes for unrecommended configuration notices in Add/Edit Config dialog

**Comment by coliu-akamai** [comment]
For the VPC create panel, just wanted to clarify if the text is only suppoed to say 'Assign this Linode to an existing VPC...' if we already have an existing vpc in that region? 

I'm seeing cases where it has the previous text - when I haven't selected any region or have selected a region that doesn't support vpcs, or a region that supports VPCs but no vpc exists in that region yet. I'm a bit unsure if this is expected behavior based on the ticket 

image

File: packages/manager/src/features/VPCs/constants.ts (hunk 1 above)
Comment (dwiley-akamai) [reply packages/manager/src/features/VPCs/constants.ts:19]: UX agreed and approved this adjustment 👍

**Comment by dwiley-akamai** [comment]
> For the VPC create panel, just wanted to clarify if the text is only suppoed to say 'Assign this Linode to an existing VPC...' if we already have an existing vpc in that region?
> 
> I'm seeing cases where it has the previous text - when I haven't selected any region or have selected a region that doesn't support vpcs, or a region that supports VPCs but no vpc exists in that region yet. I'm a bit unsure if this is expected behavior based on the ticket (M3-7499)

Checked with UX and confirmed that is the intended handling

**Review by coliu-akamai — APPROVED** [review]
awesome thanks for checking Dajahi!

- confirmed learn more link
- confirmed copies in VPC panel
- confirmed unrecommended configurations copies

**Review by bnussman-akamai — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]

**Pull request merged**